lib/sqlalchemy/orm/session.py

```diff
@@ -1633,7 +1633,8 @@ def __init__(
             .. tip:: When using SQLite, the SQLite driver included through
                Python 3.11 does not handle SAVEPOINTs correctly in all cases
                without workarounds. See the section
-               :ref:`pysqlite_serializable` for details on current workarounds.
+               :ref:`pysqlite_serializable` or :ref:`aiosqlite_serializable`
```

Comment:
**mike bayer** (zzzeek) wrote:

"and"

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4907